modules/end-user-guide/partials/proc_overriding-a-secrets-mount-path-for-a-workspace-component.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
+
+[id="overriding-a-secrets-mount-path-for-a-workspace-component_{context}"]
+= Overriding a secret's mount path for a workspace component
+
+WARNING: On 
+ifeval::["{project-context}" == "che"]
+{kubernetes} older than v1.13 and
+endif::[]
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
+
+You can override a secret's mount path for a workspace component by editing the devfile.
```

Comment:
```suggestion
You can override the mount path of a secret for a workspace component by editing the devfile.
```

Reply:
@rkratky I only added the possessive here to avoid the misunderstanding that the secret is only for one workspace component ("a secret for a workspace component").

Reply:
I think I can change this phrase throughout the file to "override/ing the mount path of the secret for a workspace component".